standalone-deploy/build_standalone_docker.sh

```diff
@@ -26,9 +26,15 @@ rm -rf init.sh requirments.txt requirements.txt
 
 cp ../requirements.txt ./docker/python
 
-#sed -i "s/'user':.*/'user': 'fate_dev',/g" ./fate_flow/settings.py
-#sed -i "s/'passwd':.*/'passwd': 'fate_dev',/g" ./fate_flow/settings.py
-#sed -i "s/'host':.*/'host': 'mysql',/g" ./fate_flow/settings.py
+if [ `uname -s` == "Darwin" ]; then
+    sed -i ".bak" "s/'user':.*/'user': 'fate_dev',/g" ./fate_flow/settings.py
```

Comment:
Thank you for your suggestion, but here can use 'sed -i "" "s/'user':.*/'user': 'fate_dev',/g" ./fate_flow/settings.py' to keep linux and mac consistent. We will open a new pull request.

Reply:
Thanks so much for resolving this problem with better solution =)